src/components/wallet/UserTransactionsTable.tsx

```diff
@@ -50,31 +50,39 @@ const getColumnsRenderers = (address: string): Record<Column, (value: any, tx: U
 
 const TransactionsTable: React.FC<Props> = ({ transactions }): JSX.Element => {
   const intl = useIntl()
+  const isDesktopView = Grid.useBreakpoint()?.lg ?? false
 
   const address = 'tbnb1vxutrxadm0utajduxfr6wd9kqfalv0dg2wnx5y'
 
   const columnsRenderers = useMemo(() => getColumnsRenderers(address), [address])
 
   return (
     <StyledTable dataSource={transactions} rowKey="_id" pagination={false}>
-      <Table.Column
-        title={intl.formatMessage({ id: 'common.type' })}
-        dataIndex="txType"
-        align="left"
-        render={columnsRenderers.type}
-      />
-      <Table.Column
-        title={intl.formatMessage({ id: 'common.address' })}
-        dataIndex="txFrom"
-        align="left"
-        render={columnsRenderers.address}
-      />
-      <Table.Column
-        title={intl.formatMessage({ id: 'common.to' })}
-        dataIndex="Type"
-        align="left"
-        render={columnsRenderers.to}
-      />
+      {isDesktopView && (
```

Comment:
hm. is there any reason to to split all of this views to separate condition renders ? 
i mean there are 3 identical checks for `isDesktop`

Reply:
@FrozenIce0617 _As a suggestion:_ Similar to `PoolsOverview` you can split columns into `mobileColumns` and `desktopColumns` . See https://github.com/thorchain/asgardex-electron/blob/32e7296709c9a1d6808b54f6635b920a43e292f3/src/views/pools/PoolsOverview.tsx#L290-L302